pkg/limayaml/defaults.go

```diff
@@ -55,6 +58,21 @@ func FillDefault(y *LimaYAML) {
 		FillPortForwardDefaults(&y.PortForwards[i])
 		// After defaults processing the singular HostPort and GuestPort values should not be used again.
 	}
+	for i := range y.Network.VDE {
+		vde := &y.Network.VDE[i]
+		if vde.MACAddress == "" {
+			// every interface in every limayaml file must get its own unique MAC address
+			uniqueID := fmt.Sprintf("%s#%d", filePath, i)
```

Comment:
We should also put host UUID to avoid MAC collision when multiple Mac hosts with bridged mode are connected to the same physical network. (Can be another PR)

https://apple.stackexchange.com/questions/342042/how-can-i-query-the-hardware-uuid-of-a-mac-programmatically-from-a-command-line

I guess we can just use hostname instead, but UUID seems more stable than hostname.

We should probably also put user ID to the hasher.

Reply:
> We should also put host UUID to avoid MAC collision when multiple Mac hosts with bridged mode are connected to the same physical network. (Can be another PR)

I've create #135 for this. Does it make sense to use VDE on Linux too?

> We should probably also put user ID to the hasher.

Why would we need this; the full path to the `lima.yaml` file is already unique for each machine. If 2 users refer to the same config file, they will refer to the same instance. Also for regular operation the filename already includes the user name: `/Users/$USER/.lima/$INSTANCE/lima.yaml`.